modules/nw-udn-support-matrix-primary-secondary.adoc

```diff
@@ -0,0 +1,53 @@
+//module included in the following assembly:
+//
+// *networkking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="support-matrix-for-primary-secondary-networks_{context}"]
+= Support matrix for primary and secondary networks
+
+The following table outlines supported features of primary and secondary networks created by `UserDefinedNetwork` (UDN) and other secondary networks. For more information on, the limitations of user-defined networks see: xref:../networking/ . For more information on secondary networks that are created through network attachement defintions using multus, see xref:../networking/.
+
+[cols="<.^3,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1"]
+|====
+|Network type |definition |role |services |east-west traffic |north-sourth traffic |Persistent IPs |`NetworkPolicy`| `MultiNetworkPolicy` |Ingress |Egress IP
+
+|UDN primary
+|A network that is created by the user using the UDN custom resource definition (CRD) and declared either primary or secondary.
```

Comment:
based on this being called UDN primary i'd expect it to be declared as primary. is that right? if not, that's confusing! 😅 
```suggestion
|A network that is created by the user using the UDN custom resource definition (CRD) and declared as primary.
```